web/handlers.go

```diff
@@ -150,7 +150,7 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Frame-Options", "SAMEORIGIN")
 		// Set content security policy. This is also specified in the root.html of the webapp in a meta tag.
 		w.Header().Set("Content-Security-Policy", fmt.Sprintf(
-			"frame-ancestors 'self'; script-src 'self' cdn.segment.com/analytics.js/%s",
+			"frame-ancestors 'self'; script-src 'self' cdn.segment.com/analytics.js/%s cdn.rudderlabs.com",
```

Comment:
@lieut-data adding you to the review specifically because of this line, which I do not really understand at all, particularly the format string being applied to the segment entry in the CSP header. Git blame reckons you added this so hoping you can check if I've done the right thing here.

Reply:
The `cspShaDirective` is an optional addition to the `script-src` directive to allow for an inline script to be injected that configures the subpath on startup. It's [complicated](https://developers.mattermost.com/blog/subpath/).

Let's keep it at the end, though:
```suggestion
			"frame-ancestors 'self'; script-src 'self' cdn.segment.com/analytics.js/ cdn.rudderlabs.com%s",
```

Reply:
Thanks for the info Jesse, I was confused by the lack of a space before the `%s` which made me think it was somehow related to the segment rule in their :man_facepalming: I'll update the PR with this fix.